src/test/resources/features/booking_activities.feature

```diff
@@ -122,6 +122,8 @@ Feature: Booking Activities
     Then bad request response, with "Invalid date range: toDate is before fromDate." message, is received from booking activities API
 
 # Pay is Nomis-only for now due to the "offender id to booking id mapping sql" using nomis-specific booking_seq column
+  # step not implemented so marked as broken
+  @broken
```

Comment:
it appears that old cucumber / serenity ignored any tests where the steps weren't implemented :facepalm: